src/lib/structures/fpa_rounding.ml

```diff
@@ -286,20 +269,22 @@ let round_to_integer mode q =
 let make_adequate_app s l ty =
   match s with
   | Sy.Name (hs, Sy.Other) when Options.get_use_fpa() ->
+    let ei i = E.int (string_of_int i) in
+    let float prec exp ?(mode = _NearestTiesToEven__rounding_mode) x =
+      Sy.(Op Float), [ei prec; ei exp; mode; x]
+    in
+    let float32 = float 24 149 in
+    let float64 = float 53 1074 in
```

Comment:
A bit pedantic but why not:
```suggestion
    let float prec exp ?(mode = _NearestTiesToEven__rounding_mode) x =
      Sy.(Op Float), [E.int prec; E.int exp; mode; x]
    in
    let float32 = float "24" "149" in
    let float64 = float "53" "1074" in
```

Reply:
This is trading a tiny bit of typing for a tiny bit of performance, and I like types. If you find it too convoluted I can use your version (I knew I should have gone all the way and defined `int' n = int (string_of_int n)` in `expr.ml` (: ).

Reply:
I agree that using the right types is more elegant :) no need to change